website/docs/r/cloud_project_database_m3db_namespace.html.markdown

```diff
@@ -0,0 +1,98 @@
+---
+layout: "ovh"
+page_title: "OVH: cloud_project_database_m3db_namespace"
+sidebar_current: "docs-ovh-resource-cloud-project-database-m3db-namespace"
+description: |-
+  Creates a namespace for a M3DB cluster associated with a public cloud project.
+---
+
+# ovh_cloud_project_database_m3db_namespace
+
+Creates a namespace for a M3DB cluster associated with a public cloud project.
+
+## Example Usage
+
+```hcl
+data "ovh_cloud_project_database" "m3db" {
+  service_name  = "XXX"
+  engine        = "m3db"
+  id            = "ZZZ"
+}
+
+resource "ovh_cloud_project_database_m3db_namespace" "namespace" {
+  service_name  = data.ovh_cloud_project_database.m3db.service_name
+  cluster_id    = data.ovh_cloud_project_database.m3db.id
+  group         = "mygroup"
```

Comment:
`retention_period_duration` instead? With an example of the required format please :)